.github/workflows/main.yml

```diff
@@ -61,3 +61,10 @@ jobs:
     with:
       version: ${{ needs.version.outputs.nextVersionTag }}
       prerelease: ${{ needs.version.outputs.isPreRelease == 'true' }}
+  cascading:
+    runs-on: ubuntu-latest
+    needs: [version, build, checks]
```

Comment:
in fact when it's merged on the release branch should we wait that it build again to cascade the change ?
or should we try to cascade directly ?

Reply:
As discussed, in case there is a cascading already done from the previous branch, better ensure that all the checks are green before cascading again